### ISSUE TITLE
TensorRt

### ISSUE THREAD

**Issue by sctrueew** [midasklr/yolov5ds#4]
@midasklr Hi,

Thank you for your work.  Is it possible to run it on TensorRT C++ or Onnx C++?

**Comment by midasklr** [comment]
of course, you can refer to https://github.com/wang-xinyu/tensorrtx/tree/master/yolov5  to add the segmentation head on the neck of yolov5.

**Comment by sctrueew** [comment]
@midasklr Hi,

Could you please some tips to do that?
